src/api/api.ts

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 import { AuthResponse } from '../types/types';
 
-export const API_URL = 'http://127.0.0.1:5000';
+export const API_URL = 'https://rsclone-backend-production.up.railway.app/';
```

Comment:
Я вчера дала не совсем верную ссылку. Правильно так:
```suggestion
export const API_URL = 'https://rsclone-backend-production.up.railway.app';
```